ex10/style.css

```diff
@@ -0,0 +1,27 @@
+div {
+  margin: 10px;
+}
+
+.input {
+  width: 250px;
+  border: 0 solid black;
+  border-width: 2px;  
+}
+
+.hide {
+  display: none;
+}
+
+ol {
+  width: 250px; 
+  border: 0 solid black;
+  border-width: 2px;
+}
+
+li {
+  border-bottom: 1px dashed black;
+}
+
+.last {
```

Comment:
Please use `:nth-last-child` (https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-last-child) instead (this will mean you will not need to remember to add a `last` class.